README.md

```diff
@@ -111,16 +111,62 @@ If you installed globally, run
 prettier path/to/file.php --write
 ```
 
+## Editor integration
+
+Integration in the prettier plugin for your favorite editor might not be working yet, see see the related issues for [VS Code](https://github.com/prettier/prettier-vscode/issues/395), [Atom](https://github.com/prettier/prettier-atom/issues/395) and [Vim](https://github.com/prettier/vim-prettier/issues/119).
+
+For the moment, you can set up prettier to run on save like this:
+
+### Atom
+
+Install [save-autorun](https://atom.io/packages/save-autorun) and create a `.save.cson` file in your project with the following content:
+
+```cson
+"**/*.php": "prettier ${path} --write"
+```
+
+### VScode
+
+Install [Run on Save](https://marketplace.visualstudio.com/items?itemName=emeraldwalk.RunOnSave) and add the following section to your settings:
+
+```json
+"emeraldwalk.runonsave": {
+  "commands": [
+    {
+      "match": "\\.php$",
+        "cmd": "prettier ${file} --write"
+    }
+  ]
+}
+```
+
+### Vim
+
+Adding the following to `.vimrc` will define a custom command `:PrettierPhp` that runs the plugin while preserving the cursor position and run it on save.
+
+```vim
+" Prettier for PHP
+function PrettierPhpCursor()
+  let save_pos = getpos(".")
+  %! prettier --stdin --parser=php
+  call setpos('.', save_pos)
+endfunction
+" define custom command
+command PrettierPhp call PrettierPhpCursor()
+" format on save
+autocmd BufwritePre *.php PrettierPhp
+```
+
 ## Contributing
 
 If you're interested in contributing to the development of Prettier for PHP, you can follow the [CONTRIBUTING guide from Prettier](https://github.com/prettier/prettier/blob/master/CONTRIBUTING.md), as it all applies to this repository too.
 
 To test it out on a PHP file:
 
-- Clone this repository.
-- Run `yarn`.
-- Create a file called `test.php`.
-- Run `yarn prettier test.php` to check the output.
+* Clone this repository.
+* Run `yarn`.
+* Create a file called `test.php`.
+* Run `yarn prettier test.php` to check the output.
```

Comment:
We need use prettier for readme :smile:

Reply:
This change was actually done by prettier :smile:

Reply:
hm, https://prettier.io/playground/#N4Igxg9gdgLgprEAuEAqABAYQDbTumACwEsBndAJzgAcJTiYIKBPAOgB0oMAlAVynQADZgEMKUQRxgBqHvyGjxkzhkxUR8dCPQAzYtnxgR2AwBMh8UjFbVC1ZTNXrN2vQfRGTcc4MvXb9hxc6HwCwmIC1FQwMMRwFARwVjZ2ggQQHoRwYADWBFnoELww1MVSKiHy4eLoUXAxcQl+Kfbpmdl5RPhFJWUgADQgENSx0KTIoGIUEADuAApiCOMoIgBuEMSmAyAARhQiufUAytQHxFAA5sgwFLxwg+ek8TBz+xcAtiLIOsZPgwBWpAAHgAhfaHGBHETvOAAGXOcG+v3uIFOFCeFGQIE+FByplmUG2UXOMAA6psiMgABwABkGUTocFJ+2oWLqGNWiMGVAAjrxiFRXiIPl8kD9sH8QE93sRrrcUfRLgYAIq8CDwJESlEwEQ7cmmSlIABMgxuIn05wumAg70+WKgeG2vCeABVdctxU8AL5eoA

Reply:
Hm, strange. Maybe I have a custom config somewhere, let me see ...

Reply:
Seems that vim-prettier is doing that. I reverted the change.

Reply:
I know what happened now. vim-prettier was running the prettier version inside our repository's `node_modules`, which is still 1.12 and not 1.13 - we should probably update it :wink: